### DISCUSSION TITLE
contact / intro fr the ELM-Python-Client

### DISCUSSION THREAD

**Discussion by rreis** [IBM/ELM-Python-Client#74] [General]
Hi

i wanted to reach out and know if the key developers for ELM would be interest in doing a presentation for our company community of practice of Sys Eng?

cheers, Ricardo

**Comment by galashour** [comment]
Also if there is someone from the dev team that can be approached with a beginner question related to ewm/ccm - would be appreciated

**Comment by rsjazz** [comment]
This is an issue channel for ELM-Python-Client, this is not a contact IBM channel. 

If you have questions you can try and ask in https://jazz.net/forum .
If you would like to request talks, you could also ask there. Or you approach your IBM sales contact and ask them, if they can arrange something.
